### PR TITLE
Skip processing animated gifs

### PR DESCRIPTION
**Pull request by joehoyle**
If gifs are animated, then don't try to do anythign with them as we'll
be discarding the animation.

### CODE REVIEW

**Comment by roborourke** [comment]
@joehoyle looks, good, do you need me to test it too?

**Comment by joehoyle** [comment]
@roborourke if that's ok, would be great!

File: index.js [hunk 1]
```diff
@@ -46,7 +47,13 @@ module.exports.resizeBuffer = function( buffer, args, callback ) {
 
 			// convert gifs to pngs
 			if( path.extname( args.key ).toLowerCase() === '.gif' ) {
-				image.png()
+
+				if ( isAnimated( buffer ) ) {
+					return callback( null, buffer, { size: buffer.length, format : 'gif' } )
+				} else {
+					image.png()
+				}
```
Comment (rmccue) [inline index.js]: Unnecessary else

**Comment by roborourke** [comment]
@joehoyle does the job, cheers :)

**Pull request merged**